sycl-fusion/common/include/Kernel.h

```diff
@@ -61,6 +61,39 @@ enum class ParameterKind : uint32_t {
 /// Different binary formats supported as input to the JIT compiler.
 enum class BinaryFormat : uint32_t { INVALID, LLVM, SPIRV, PTX, AMDGCN };
 
+/// Unique ID for each supported architecture in the SYCL implementation.
+using DeviceArchitecture = unsigned;
+
+/// ID common to every SPIR-V target architecture.
+///
+/// As an exception, SPIR-V architectures have a single common ID, as fused
```

Comment:
What does `SPIR-V architectures` mean? There is no such term. And SPIR-V is device-agnostic. Can it be re-written?

Reply:
I rephrased and renamed. LG now?